web/pandas/pdeps/0006-ban-upcasting.md

```diff
@@ -216,13 +225,15 @@ at all. To keep this proposal focused, it is intentionally excluded from the sco
 
 **Q: What if I want ``1.0000000001`` to be inserted as ``1.0`` in an `'int8'` Series?**
 
-**A**: You may want to define your own helper function, such as
+**A**: You may want to define your own helper function, such as:
+
   ```python
-  >>> def maybe_convert_to_int(x: int | float, tolerance: float):
+  def maybe_convert_to_int(x: int | float, tolerance: float):
       if np.abs(x - round(x)) < tolerance:
           return round(x)
       return x
   ```
+
```

Comment:
likewise

Reply:
The FAQ source code block should render correctly now, and I will resolve upon full code check and preview pass.

Reply:
sure, thanks - just fyi there's no need to merge upstream/main unless you have to (e.g. if there's conflicts, or if it's an old PR)

Reply:
Local build has indicated proper code block formating for this code block in the FAQ.